src/vl-progress-bar-item.js

```diff
@@ -0,0 +1,59 @@
+import { VlElement, define } from '/node_modules/vl-ui-core/dist/vl-core.js';
+
+/**
+ * VlProgressBarItem
+ * @class
+ * @classdesc Onderdeel van de progress-bar die een stap voorstelt van een proces dat uit verschillende stappen bestaat.
+ * 
+ * @extends VlElement
+ * 
+ * @property {boolean} data-vl-active - Attribuut wordt gebruikt om aan te geven dat de stap actief is.
+ * 
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-progress-bar/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-progress-bar/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-progress-bar-item.html|Demo}
+ * 
+ */
+export class VlProgressBarItem extends VlElement(HTMLElement) {
+    /**
+     * Geeft de tekst van de progress bar item.
+     * 
+     * @return {string}
+     */
+    get text() {
+        return this.innerText;
+    }
+
+    /**
+     * Geeft terug of de progress bar item actief is.
+     * 
+     * @return {boolean}
+     */
+    get active() {
+        return this.hasAttribute('data-vl-active');
+    }
+
+    get template() {
+        const activeClass = this.active ? 'vl-progress-bar__step--active' : '';
+        const ariaCurrent = this.active ? 'aria-current="step"' : '';
+        return this._template(`
+            <div class="vl-progress-bar__step ${activeClass}">
+                <button class="vl-progress-bar__bullet" ${ariaCurrent} aria-label="${this.text}">
```

Comment:
moet geen vl-button zijn?

Reply:
Nee, heeft bij Webuniversum niet de vl-button classes, dus is niet de vl-button. Anders zou de stijl verkeerd zijn wanneer dit wel toegepast zou worden.